src/components/labware/PacbioTubeWell.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
+    <div :class="wellClassNames" data-attribute="traction-well" @click="click">
+      <p v-if="position" data-attribute="traction-well-position" class="truncate font-light">
+        {{ position }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script setup>
+/**
+ * @name PacbioTubeWell
+ * @description Renders a single well (can be used in a Tube) without any store interaction
+ * @param {Array} requests - An array of request ids associated with the well
+ * @param {String} position - The position of the well
+ * @emits click - Emits the request id and selected status of the well
+ */
+import { computed } from 'vue'
+
+const props = defineProps({
+  /**
+   * An array of request ids associated with the well
+   */
+  requests: {
+    type: Object,
+    required: true,
+  },
+  /*
+   * The position of the well
+   * If the position is provided, it will be displayed in the well
+   */
+  position: {
```

Comment:
It is not currently in use, but it is kept there in case it is required for any future context with a position text displayed. However, yes, it can be removed. 👍